doc/conf.py

```diff
@@ -48,17 +48,18 @@
 
 # General information about the project.
 project = 'ephyviewer'
-copyright = '2017, Samuel Garcia'
-author = 'Samuel Garcia'
+copyright = '2019, Samuel Garcia, Jeffrey Gill'
+author = 'Samuel Garcia, Jeffrey Gill'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
+exec(open('../ephyviewer/version.py').read())
 # The short X.Y version.
-version = '1.0'
+version = version
```

Comment:
I don't remember why but I think this is better:
```
d = {}
exec(open("spiketoolkit/version.py").read(), None, d)
version = d['version']
```

Reply:
Ah yes, I could see why that is. Your way ensures the global namespace isn't polluted with other things. For example, if another variable was added to `version.py`, my way would read it in too, and it could potentially overwrite something. I'll change it.